full/src/main/java/apoc/load/LoadDirectory.java

```diff
@@ -0,0 +1,55 @@
+package apoc.load;
+
+import apoc.Extended;
+import apoc.result.StringResult;
+import apoc.util.FileUtils;
+import org.apache.commons.io.filefilter.TrueFileFilter;
+import org.apache.commons.io.filefilter.WildcardFileFilter;
+import org.neo4j.logging.Log;
+import org.neo4j.procedure.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.*;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static apoc.ApocConfig.apocConfig;
+
+@Extended
+public class LoadDirectory {
+
+    @Context
+    public Log log;
+
+    @Procedure
+    @Description("apoc.load.directory('pattern', 'urlDir', {config}) YIELD value - List of all files in folder specified by urlDir or in import folder if urlDir string is empty or not specified")
+    public Stream<StringResult> directory(@Name(value = "pattern", defaultValue = "*") String pattern, @Name(value = "urlDir", defaultValue = "") String urlDir, @Name(value = "config", defaultValue = "{}") Map<String, Object> config) throws IOException {
+        log.info("Search files that match regular expression: " + pattern);
+
+        if (urlDir == null)
```

Comment:
```suggestion
        if (StringUtils.isBlank(urlDir))
```

Reply:
Qui la mia idea era mettere la stringa vuota `""` per dire alla procedura di vedere i file nella cartella di import, mentre ad esempio mettendo `"subfolder"` vedrei i file della sottocartella degli import, similmente con le altre procedure in cui per caricare un file nella cartella di import farei ad esempio `apoc.load.json('file.json')` mentre per caricare un file in una sottocartella farei `apoc.load.json('subfolder/file.json')`. 
Se impedisco di passare la stringa vuota, non potrei fare la stessa cosa analogamente alle altre procedure,
e per vedere i file nella cartella di import dovrei per forza impostare `apoc.import.file.use_neo4j_config=false` e poi passare `file:///urlCartellaImport` . O sbaglio qualcosa?

Reply:
va bene aggiungilo nella documentazione ok?